src/Command/RandomIssueCommand.php

```diff
@@ -59,6 +56,21 @@ protected function configure(): void
             InputOption::VALUE_OPTIONAL,
             'Do you want issues for #hacktoberfest?'
         );
+
+        $this->addOption(
+            'source',
+            's',
+            InputOption::VALUE_OPTIONAL,
```

Comment:
I think the value should be required if `source` is given as an option. Since there is no point in just providing `--source` if its still github as default. So `InputOption::VALUE_REQUIRED` should be set.